### ISSUE TITLE
Bringing back the old sync buttom

### ISSUE THREAD

**Issue by GwnMichael** [xgi/houdoku#393]
### Acknowledgements

- [X] Check that there is not an existing ticket for this request, and that it is not related to an existing ticket.
- [X] Create one ticket per feature. Feel free to create multiple tickets.
- [X] If this is an extension-related request, open a ticket in the [extensions repo](https://github.com/xgi/houdoku-extensions/issues/new/choose) instead.
- [X] Write a concise title for this ticket in the field above.

### Feature details

Could it be possible to bring back the old sync buttom on top again what checked all the chapters for you, instead of manual going by every chapter and checking them yourself?

![ewgerger12](https://github.com/user-attachments/assets/2acf2c01-9cde-4bc0-976c-4385173d9f7d)

**Comment by ojvaj1** [comment]
Yes i fully agree. I really love the ui update but when i was going to read i was missing the button. Bring it back pls!
